userdata/adam_zone_per_device/core.appliances.xml

```diff
@@ -86,7 +86,7 @@
 				<unit></unit>
 				<last_consecutive_log_date>2020-03-20T12:15:29.245+01:00</last_consecutive_log_date>
 				<period start_date="2020-03-20T12:15:29.245+01:00" end_date="2020-03-20T12:15:29.245+01:00">
-					<measurement log_date="2020-03-20T12:15:29.245+01:00">1.00</measurement>
+					<measurement log_date="2020-03-20T12:15:29.245+01:00">0.00</measurement>
```

Comment:
Maybe cross-check with HA's tests - otherwise we shouldn't forget (in general) to update our fixtures from testing to a new HA-PR as well.